### DISCUSSION TITLE
Package Suggestion: Cloudflare WARP

### DISCUSSION THREAD

**Discussion by rafay-pk** [ScoopInstaller/Main#4776] [Ideas]
It's a free to use dns resolver + vpn based on cloudflares infrastructure. 
You can find the releases on: https://1.1.1.1/
Other package managers like chocolatey have it: https://community.chocolatey.org/packages/warp

**Comment by HUMORCE** [comment]
https://github.com/ScoopInstaller/Extras/issues/10810
